src/plugins/data/common/search/expressions/esql.ts

```diff
@@ -156,14 +162,17 @@ export const getEsqlFn = ({ getStartDependencies }: EsqlFnArguments) => {
             }
 
             const request = inspectorAdapters.requests.start(
-              i18n.translate('data.search.dataRequest.title', {
-                defaultMessage: 'Data',
-              }),
-              {
-                description: i18n.translate('data.search.es_search.dataRequest.description', {
-                  defaultMessage:
-                    'This request queries Elasticsearch to fetch the data for the visualization.',
+              titleForInspector ??
+                i18n.translate('data.search.dataRequest.title', {
+                  defaultMessage: 'Data',
```

Comment:
Can we call them: `Visualization` and `ESQL`?  it feels a bit more clear, because Data and Results are completely vague in this context. WDYT?

Also because the label is prefixed by `Request:` so it will read by `Request: Visualization` and `Request: ESQL` that to me is pretty clear what is going on.
With `Request: Data` and `Request: Results` are very very vague, even if there is a description down below